Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -1,31 +1,34 @@
 # L5: Principal Software Engineer
 
-> _I am an influential Engineering leader who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus._
+> _I am an influential Engineering leader who identifies problems and opportunities relevant to Octopus product and technical strategies, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus._
 
-- **Planning horizon**: 1+ year
-- **Impact radius**: Group+ (15-50). Note: This role is evolving as the company grows. PSE is right at the top of our IC track. For example, the impact radius expectations of an L5 will likely grow as Octopus grows larger over the next couple years, from 2-4 teams in 2021. We expect the role to stabilise in 2022 once the company has grown to the point of needing an L6 IC engineering role.
-- **Evaluation**: Director or VP
-- **Responsibility and direction needed**: Drives technical strategy in alignment with product strategy and drives high-impact improvements across teams.
+- **Thinking horizon**: 1 year
+- **Impact radius**: Group
+- **Evaluation**: Director
+- **Responsibility and direction needed**: I drive technical strategy in alignment with product strategy for my group. Point me in the right direction with helpful constraints and good outcomes happen with minimal oversight.
 
 ## 🦉 Domain expertise
 
-- I contribute to core technologies used by a variety of teams.
-- I help teams make critical product architecture and implementation decisions.
-- I maintain a vision of the overall technical direction within my impact radius.
-- I contribute to the engineering standards and principles of the company.
-- I contribute to the Octopus technical strategy.
-- I manage technical risks, and architect to reduce technical risk.
+- I maintain a clear and understandable vision for the technical direction for teams within my group.
+- I have a deep understanding of the technical strategy and target architecture aligned with my group.
+- I help teams in my group make critical product architecture and implementation decisions that result in better outcomes.
+- I have a broad understanding of the Octopus technical strategy and target architecture.
```

Comment:
Nit: I'd like the "I have broad understaing..." to be just above "I have deep understanding...", similar to the customer success section, as the information flows well that way.